src/views/creating.handlebars

```diff
@@ -97,12 +97,9 @@
     <div id="maindiv">
       <h1 id="title">Creating Wi-Fi network...</h1>
       <h3 id="content">
-        Please connect to {{ap_ssid}} with the password you just created.
+        Please connect to {{ap_ssid}} with the password you just created, then
+        navigate to {{domain}}.local or {{ap_ip}} in your web browser.
```

Comment:
A minor QoL improvement here would be to make these links. Although I think many implementations of captive portal support close the page faster than a user can click them

Reply:
Force pushed an update with links.